Airport/Airport.js

```diff
@@ -0,0 +1,92 @@
+const { PassengerPlane, MilitaryPlane, ExperimentalPlane } = require('../Planes');
+
+class Airport {
+  constructor(planes) {
+    this._planes = planes;
+  }
+
+  get planes() {
+    return this._planes;
+  }
+
+  static print(planes) {
+    return JSON.stringify(planes);
+  }
+
+  sortByMaxDistance() {
+    this.planes.sort((a, b) => (a.maxFlightDistance > b.maxFlightDistance ? 1 : -1));
+    return this;
+  }
+
+  sortByMaxSpeed() {
+    this.planes.sort((a, b) => (a.maxSpeed > b.maxSpeed ? 1 : -1));
+    return this;
+  }
+
+  sortByMaxLoadCapacity() {
+    this.planes.sort((a, b) => (a.maxLoadCapacity > b.maxLoadCapacity ? 1 : -1));
+    return this;
+  }
+
+  getPlanesByPurpose(purpose) {
+    const wantedPlanesArray = [];
+    try {
+      switch (purpose) {
+        case 'Passenger':
+          this.planes.forEach((plane) => {
+            if (plane instanceof PassengerPlane) {
+              wantedPlanesArray.push(plane);
+            }
+          });
+          break;
+        case 'Military':
+          this.planes.forEach((plane) => {
+            if (plane instanceof MilitaryPlane) {
+              wantedPlanesArray.push(plane);
+            }
+          });
+          break;
+        case 'Experimental':
+          this.planes.forEach((plane) => {
+            if (plane instanceof ExperimentalPlane) {
+              wantedPlanesArray.push(plane);
+            }
+          });
+          break;
+        default:
+          throw new Error(
+            'Sorry, no such plane type exists. ' + 'Please try one of the following: Passenger, Military, Experimental'
+          );
+          break;
+      }
+    } catch (e) {
+      console.log(e);
+    }
+    return wantedPlanesArray;
+  }
+
+  getMilitaryPlanesByType(type) {
```

Comment:
that is actually why the method above is called getPlanesByPurpose.
Since plane of some Purposes Military,Experimental) has types(e.g. Military Types and Experimental Types)

Reply:
Moving this method to the MilitaryPlane class will resolve unclarity.